src/toxext.h

```diff
@@ -23,7 +23,7 @@ struct ToxExtPacketList;
 
 #define TOXEXT_MAX_PACKET_SIZE (TOX_MAX_CUSTOM_PACKET_SIZE - 7)
 
-enum TOXEXT_ERROR {
+enum Toxext_Error {
```

Comment:
I see, this is cause the tox core coding standard changed and we want to match

Reply:
lol yeah didn't mean to stomp your coding style. Just noticed that toxcore has defined CAP_ENUMS but then typedefs to Camel_Enums for the types, and has deprecated the originals, and thought you might want to have the same.